setup.cfg

```diff
@@ -22,7 +22,7 @@ filterwarnings =
 
 [flake8]
 max-line-length = 88
-ignore = E203, E266, E501, W503
+ignore = E203, E266, E501, W503, F821
```

Comment:
I wonder if a comment could be placed here that explains _what_ we ignore

Reply:
It should be possible. Can we do it in a separate PR?

Reply:
Sure, comments here are orthogonal to this PR.